lib/github_changelog_generator/generator/generator_processor.rb

```diff
@@ -156,7 +156,7 @@ def include_issues_by_labels(issues)
       filter_wo_labels(filtered_issues)
     end
 
-    # @param [Array] issues Issues & PRs to filter when without labels
+    # @param [Array] items Issues & PRs to filter when without labels
```

Comment:
It's debatable whether this commit belongs in this PR.
I would probably prefer a Rubocop cop or something that auto-corrects this.

Reply:
Yard-junk is a tool for this. Finding errors and presenting them in a small report.